src/data_morph/shapes/bases/line_collection.py

```diff
@@ -23,6 +23,11 @@ class LineCollection(Shape):
     """
 
     def __init__(self, *lines: Iterable[Iterable[Number]]) -> None:
+        # check that lines that have the same starting and ending points raise an error
+        for line in lines:
+            start, end = line
+            if np.allclose(start, end):
+                raise ValueError(f'Line {line} has the same start and end point')
         self.lines = lines
```

Comment:
I think that change would break this test here:
https://github.com/stefmolin/data-morph/blob/4d5319d1f42bbeea765832884f19421efc9ba6af/tests/shapes/bases/test_line_collection.py#L43
Should I remove it (the test I mean, not the change)?

Reply:
Let's update the test to reflect the change.